roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install RPM key for StackStorm
+  become: yes
+  rpm_key:
+    state: present
+    key: https://packagecloud.io/StackStorm/stable/gpgkey
+
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: stackstorm
+    description: Stackstorm Repo
+    file: st2
```

Comment:
So would you prefer I undo https://github.com/StackStorm/ansible-st2/pull/66/commits/8f1e76b94b974379ead556b67f8fec2ec170338b? I was using the script to automatically generate this configuration previously